sql-to-dbsp-compiler/SQL-compiler/src/main/java/org/dbsp/sqlCompiler/compiler/frontend/statements/CreateRelationStatement.java

```diff
@@ -128,10 +128,16 @@ public JsonNode getDefinedObjectSchema() {
         ObjectNode result = mapper.createObjectNode();
         result.put("name", this.relationName);
         ArrayNode fields = result.putArray("fields");
+        ArrayNode keyFields = mapper.createArrayNode();
+        boolean hasKey = false;
         for (RelColumnMetadata col: this.columns) {
             ObjectNode column = fields.addObject();
             column.put("name", col.getName());
             column.put("case_sensitive", col.nameIsQuoted);
+            if (col.isPrimaryKey) {
```

Comment:
Will this work if the primary key is declared as a separate clause (`primary key (col1, col2)`)?

Reply:
yes, the metadata is the same.